tests/performance/plugins/server-timing/mu-plugin.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+add_action(
+	'template_redirect',
+	function() {
+		ob_start();
+
+		global $server_timing_values, $timestart;
+
+		if ( ! is_array( $server_timing_values ) ) {
+			$server_timing_values = array();
+		}
+
+		$server_timing_values['before-template'] = microtime( true ) - $timestart;
+
```

Comment:
Nit-pick, but I'd advise to start the output buffer at the end of this, right here.